ethereum_history_api/oracles/src/noir/oracles/transactionOracle/encode.test.ts

```diff
@@ -0,0 +1,47 @@
+import { describe, expect, it } from 'vitest';
+import { TX_OFFSETS, encodeTx } from './encode.js';
+import { loadTxFixture } from '../../../fixtures.js';
+import { TRANSFER_TX_HASH, CHAIN_LINK_TRANSFER_TX_HASH } from '../../../fixtures/config.js';
+
+describe('TransactionOracle encode', () => {
+  describe('encodeTx', () => {
+    it('transaction with empty data', async () => {
+      const contractCreationTransaction = await loadTxFixture('mainnet', 'cancun', 'transfer', TRANSFER_TX_HASH);
```

Comment:
Now I see. This TX has empty data. Let's name the fixture `eth_transfer` than because transfer can describe both ERC20 transfer that has data as well as ETH transfer that does not